packages/mobile/src/components/core/Tile/Tile.tsx

```diff
@@ -114,21 +116,23 @@ export const Tile = <
       <TileShadow
         offset={[0, 0]}
         distance={1}
-        startColor='rgba(133, 129, 153, 0.1)'
+        // Approximately 10% opacity on neutral
+        startColor={themeColors.neutral + '1a'}
```

Comment:
nice. we should chat with julian sometime about how to do shadows in an easier way for us

Reply:
yea i agree. should maybe have specific colors built into the themes for the shadows, or a helper function to extract the shadow values from the theme. something like that